css-basic/css-basics_social-card/css/styles.css

```diff
@@ -1,3 +1,37 @@
 * {
   box-sizing: border-box;
+  font-family: sans-serif;
+  /* border: solid red 1px; */
+}
+
+body {
+  background-color: lavender;
+}
+
+img {
+  border-radius: 10px;
+}
+
+article {
+  background-color: white;
+  padding: 20px;
+  border-radius: 20px;
+}
+
+h2 {
+  text-transform: uppercase;
+}
+
+.invite {
+  font-size: 1.2rem;
+}
+
+.account {
+  color: rgb(100, 123, 153);
+}
+
+button {
+  background-color: white;
```

Comment:
Here I would use background: none instead of background-color: white, once excluding the background would make it more flexible, in case the color of the background changes in the future. But it's a minor thing.